### DISCUSSION TITLE
isAsyncWidget and initializeAsync is no longer Widget

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#5619] [Help] [answered]
Hello
I am upgrading to net 5

`isAsyncWidget `and `initializeAsync `is no longer Widget, did we update it to new one or anything

![image](https://user-images.githubusercontent.com/2279508/114098481-96393480-98eb-11eb-93b4-3040d96ff651.png)

**Comment by VictorTomaili** [comment, marked as answer]
Async widgets are removed while porting from .net to .net core. Use non async variants instead. 
`AsyncLookup` etc. still exist.

**Reply by minhhungit** [reply]
thank you
